.eslintrc.js

```diff
@@ -17,21 +16,20 @@ module.exports = {
       "flowtype"
     ],
     "rules": {
-        "indent": [
-            "error",
-            2
-        ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
+      "no-magic-numbers": [
+        "off"
```

Comment:
OK. sounds fine. If it was just a few files, I'd suggest doing control comments in the top of the file.